src/runner/api/itemRunner.js

```diff
@@ -524,6 +524,13 @@ const itemRunnerFactory = function itemRunnerFactory(providerName, data = {}, op
         isClosed(){
             return closed;
         },
+        /**
+         * Is provider cleared
+         * @return {boolean}
+         */
+        isCleared(){
```

Comment:
**thought:** I'd like to challenge this new API. What is the final purpose of it? The name may be confusing. After having a look at the implementation, I wonder what is the benefit of it, and if it is accurate with the item runner life-cycle